Makefile

```diff
@@ -76,14 +67,21 @@ test-integration:
 						-zmqpubrawtxwithfee="tcp://${TEST_INTEGRATION_ZMQ_HOST}:${TEST_INTEGRATION_ZMQ_PORT}" \
 						-fallbackfee=0.00001 \
 						-debug=rpc
-
+test-bitcoind-stop:
+				@echo "bitcoind shutdown initiated..."
+				@docker kill ${TEST_INTEGRATION_DOCKER_CONTAINER_NAME}
+				@echo "bitcoind stopped"
+				echo "removing $(TEST_INTEGRATION_DIR)"
+				rm -rf $(TEST_INTEGRATION_DIR)
```

Comment:
Should happen at the end of the function `stop_bitcoind` in the `test-integration` make-target.